### PR TITLE
restructure ml overview website page

### PR DESCRIPTION
**Pull request by Juta**
Restructure ML overview website page and add explanation about model validation

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Juta** [comment]
@damccorm Hi Danny, I made an additional edit to update the overview page of ML and to include model validation. Please have a look

File: website/www/site/content/en/documentation/ml/overview.md [hunk 1]
```diff
@@ -78,20 +80,28 @@ The RunInference API doesn't currently support making remote inference calls usi
 
 * Consider monitoring and measuring the performance of a pipeline when deploying, because monitoring can provide insight into the status and health of the application.
 
+## Model validation
+
+Model validation allows you to benchmark your model’s performance against an unseen dataset. You can extract chosen metrics, create visualizations, log metadata, and compare the performance of different models with the end goal of validating whether your model is ready to deploy. Beam provides support for running model evaluation on a TensorFlow model directly inside your pipeline.
```
Comment (damccorm) [inline website/www/site/content/en/documentation/ml/overview.md:85]:
```suggestion
Model validation allows you to benchmark your model’s performance against a previously unseen dataset. You can extract chosen metrics, create visualizations, log metadata, and compare the performance of different models with the end goal of validating whether your model is ready to deploy. Beam provides support for running model evaluation on a TensorFlow model directly inside your pipeline.
```

Small wording nit

**Review by damccorm — COMMENTED** [review]
Could you please add the license to the image file (see https://github.com/apache/beam/blob/master/website/www/site/static/images/ml-workflows.svg?short_path=6c34014 for example)

**Comment by Juta** [comment]
Thanks @damccorm. I made the requested changes

**Comment by damccorm** [comment]
(comment is for bot)

**Comment by damccorm** [comment]
Run RAT PreCommit

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**